### DISCUSSION TITLE
readme中提到的防xss攻击是如何实现，只找到了配置，但是没有找到使用配置的地方

### DISCUSSION THREAD

**Discussion by miaopass-future** [zuihou/zuihou-admin-cloud#298] [02.提问]
如题

**Comment by zuihou** [comment]
<img width="955" alt="image" src="https://github.com/dromara/lamp-cloud/assets/8500410/33d9e902-59fa-4481-9b58-eddfa1c051bf">

**Reply by miaopass-future** [reply]
找到了，在[lamp-util](https://github.com/zuihou/lamp-util)实现的。防sql注入是有特别操作么，没有找到相关的配置和代码

**Reply by zuihou** [reply]
sql 注入，在xml中使用 #{}  ，慎用 ${}
